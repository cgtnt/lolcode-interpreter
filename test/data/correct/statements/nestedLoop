HAI

IM IN YR I HAS A COUNT ITZ 1, WILE DIFFRINT COUNT AN BIGGR OF COUNT AN 5 
  IM IN YR I HAS A COUNTWO ITZ 1, WILE DIFFRINT COUNTWO AN BIGGR OF COUNTWO AN 5 
    VISIBLE SMOOSH "COUNT IZ " AN COUNT AN " " AN COUNTWO MKAY!
    COUNTWO R SUM OF COUNTWO AN 1
  IM OUTTA YR
  COUNT R SUM OF COUNT AN 1
IM OUTTA YR

VISIBLE "DONE!"

KTHXBYE
