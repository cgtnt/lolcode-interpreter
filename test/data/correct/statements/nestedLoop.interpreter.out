COUNT IZ 1 1
COUNT IZ 1 2
COUNT IZ 1 3
COUNT IZ 1 4
COUNT IZ 2 1
COUNT IZ 2 2
COUNT IZ 2 3
COUNT IZ 2 4
COUNT IZ 3 1
COUNT IZ 3 2
COUNT IZ 3 3
COUNT IZ 3 4
COUNT IZ 4 1
COUNT IZ 4 2
COUNT IZ 4 3
COUNT IZ 4 4
DONE!